web/src/components/tree/KotsApplicationTree.tsx

```diff
@@ -12,9 +12,28 @@ import FileTree from "../shared/FileTree";
 
 import "../../scss/components/troubleshoot/FileTree.scss";
 
-class KotsApplicationTree extends React.Component {
-  constructor() {
-    super();
+// Types
+import { App, KotsParams } from "@types";
+import { RouteComponentProps } from "react-router-dom";
+
+type Props = {
+  app: App;
+  appName: string;
+  appNameSpace: string;
+  isHelmManaged: boolean;
+} & RouteComponentProps<KotsParams>;
```

Comment:
going back and forth on if i want to just define `Props` and create the intersection `&` here are in the Component definition